pkg/ingester/ingester.go

```diff
@@ -74,7 +74,7 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	cfg.LifecyclerConfig.RegisterFlags(f)
 
 	f.IntVar(&cfg.MaxTransferRetries, "ingester.max-transfer-retries", 10, "Number of times to try and transfer chunks before falling back to flushing. If set to 0 or negative value, transfers are disabled.")
-	f.IntVar(&cfg.ConcurrentFlushes, "ingester.concurrent-flushed", 16, "")
+	f.IntVar(&cfg.ConcurrentFlushes, "ingester.concurrent-flushes", 16, "")
```

Comment:
I'm reluctant to change a config as this is a breaking change for anyone using this.  I agree with you that the `flushes` makes more sense but i'm inclined to just leave it as is.

Reply:
On further thought my opinion is changing here a bit, see my note on the PR itself